translated gguf.md into chinese

blueingman opened the pull request:
# What does this PR do?
I am new to the open-source community, but I’m very eager to contribute to the Transformers project. I have translated the gguf.md file into Chinese. If there are any areas that need improvement or if I overlooked anything, please let me know


## Before submitting
- [y] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [y] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

blueingman committed:
translated gguf.md into chinese

LysandreJik commented:
Nice! Maybe @Isotr0py would you be willing to review this PR? :)

Isotr0py commented:
OK, I will take a look tonight. :)

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+该文件格式是一种“单文件格式”，通常单个文件就包含了配置属性、分词器词汇表和其他属性，同时还有模型中要加载的所有张量。这些文件根据文件的量化类型有不同的格式。我们在[这里](https://huggingface.co/docs/hub/en/gguf#quantization-types)简要介绍了一些。
+
+## 在Transformers中的支持
```
```suggestion
## 在 Transformers 中的支持
```

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+### 支持的量化类型
+
+初始支持的量化类型是根据在 Hub 上分享的较为流行的量化文件决定的。
```
```suggestion
根据分享在 Hub 上的较为流行的量化文件，初步支持以下量化类型：
```

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+## 使用示例
+
+为了在`transformers`中加载`gguf`文件，你需要在 `from_pretrained`方法中为分词器和模型指定 `gguf_file`参数。下面是加载分词器和模型的示例，它们可以从同一个文件中加载：
```
```suggestion
为了在`transformers`中加载`gguf`文件，你需要在 `from_pretrained`方法中为分词器和模型指定 `gguf_file`参数。下面是从同一个文件中加载分词器和模型的示例：
```

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+### 支持的模型架构
+
+目前支持一些在Hub上非常流行的架构，即：
```
```suggestion
目前支持以下在 Hub 上非常流行的模型架构：
```

Isotr0py commented on docs/source/zh/gguf.md:
````diff
@@ -0,0 +1,104 @@
+model = AutoModelForCausalLM.from_pretrained(model_id, gguf_file=filename)
+```
+
+现在，你就已经可以结合许多其他工具,在 PyTorch 生态系统中访问完整的、未量化的模型了。
````
```suggestion
现在，你就已经可以结合 PyTorch 生态系统中的一系列其他工具，来使用完整的、未量化的模型了。
```

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+该文件格式[由抱抱脸支持](https://huggingface.co/docs/hub/en/gguf)，可用于快速检查文件中张量和元数据。
+
+该文件格式是一种“单文件格式”，通常单个文件就包含了配置属性、分词器词汇表和其他属性，同时还有模型中要加载的所有张量。这些文件根据文件的量化类型有不同的格式。我们在[这里](https://huggingface.co/docs/hub/en/gguf#quantization-types)简要介绍了一些。
```
```suggestion
该文件格式是一种“单文件格式”，通常单个文件就包含了配置属性、分词器词汇表和其他属性，同时还有模型中要加载的所有张量。这些文件根据文件的量化类型有不同的格式。我们在[这里](https://huggingface.co/docs/hub/en/gguf#quantization-types)进行了简要介绍。
```

Isotr0py reviewed (commented):
Thanks for your contributions! The translation overall LGTM. Just leave some nits to make sentence more coherent.

blueingman committed:
Apply suggestions from code review

I have updated the PR accordingly.Thank you very much for detailed guidance,and I 'll pay more attention to the details next time.

Co-authored-by: Isotr0py <2037008807@qq.com>

blueingman commented:
I have updated the PR accordingly.Thank you very much for detailed guidance,and I 'll pay more attention to the details next time. @Isotr0py @LysandreJik

Isotr0py commented on docs/source/zh/_toctree.yml:
```diff
@@ -50,6 +50,8 @@
   - local: torchscript
     title: 导出为 TorchScript
+  - local: gguf
+    title: 与 GGUF 格式共享模型
```
```suggestion
    title: 与 GGUF 格式的互操作性
```
The "Interoperability" should be translated to "互操作性" correctly, since it's a computer terminology.

Isotr0py reviewed (approved):
Just a minor mistake. Otherwise LGTM.

Isotr0py commented on docs/source/zh/gguf.md:
```diff
@@ -0,0 +1,104 @@
+
+现在，你就已经可以结合 PyTorch 生态系统中的一系列其他工具，来使用完整的、未量化的模型了。
+
+为了将模型转换回`gguf`文件，我们建议使用`llama.cpp`中的[`convert-hf-to-gguf.py`文件](https://github.com/ggerganov/llama.cpp/blob/master/convert_hf_to_gguf.py)文件。
```
```suggestion
为了将模型转换回`gguf`文件，我们建议使用`llama.cpp`中的[`convert-hf-to-gguf.py`文件](https://github.com/ggerganov/llama.cpp/blob/master/convert_hf_to_gguf.py)。
```

blueingman committed:
Apply suggestions from code review

Co-authored-by: Isotr0py <2037008807@qq.com>

blueingman commented:
Thank you again for your careful suggestion!

blueingman committed:
Merge remote-tracking branch 'upstream/main' into translated_into_zh

blueingman committed:
Merge branch 'translated_into_zh' of https://github.com/blueingman/transformers into translated_into_zh

blueingman commented:
Hi @Isotr0py ,hope you're doing well. I have addressed the changes you suggested and updated the PR accordingly. When you have a moment, could you please review the updated code? Thank you for your time!

Isotr0py commented:
The updated docs LGTM. :)

Also cc @LysandreJik

LysandreJik reviewed (approved):
Thanks a lot!
